mobile-app/app/screens/AppNavigator/screens/Portfolio/components/SepaInstantLayover.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { Platform, StatusBar, View } from 'react-native'
+import { ThemedProps, ThemedTextBasic, ThemedView } from '@components/themed'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import Popover, { PopoverPlacement } from 'react-native-popover-view'
+import { Button } from '@components/Button'
+import { InfoText } from '@components/InfoText'
+import { SepaInstantComponent } from './SepaInstantComponent'
+
+export interface LayoverProps extends ThemedProps {
+  onDismiss?: () => void
+}
+
+export interface LayoverInternalProps extends LayoverProps {
+  type: LayoverType
+}
+
+enum LayoverType {
+  SepaInstantOverlay,
+  BuySuccessOverlay
+}
+
+export function SepaInstantOverlay (props: LayoverProps): JSX.Element {
+  return Overlay({ ...props, type: LayoverType.SepaInstantOverlay })
+}
+
+export function BuySuccessOverlay (props: LayoverProps): JSX.Element {
+  return Overlay({ ...props, type: LayoverType.BuySuccessOverlay })
+}
+
+export function Overlay (props: LayoverInternalProps): JSX.Element {
+  const offsetAndroidHeight = StatusBar.currentHeight !== undefined ? (StatusBar.currentHeight * -1) : 0
+  const [showPopover, setShowPopover] = useState(true)
+
+  // to fix memory leak error
+  useEffect(() => {
+    // May work on Web, but not officially supported, as per documentation, add condition to hide popover/tooltip
+    if (Platform.OS === 'web') {
+      setTimeout(() => setShowPopover(false), 2000)
+    }
+  }, [showPopover])
+
+  const onDismissInternal = (): void => {
+    setShowPopover(false)
+    props.onDismiss?.()
+  }
+
+  return (
+    <Popover
+      verticalOffset={Platform.OS === 'android' ? offsetAndroidHeight : 0} // to correct tooltip position on android
```

Comment:
I would move that OS check into the `offsetAndroidHeight` variable. Because then it either has `-StatusBar.currentHeight` or `0` and the name would be more clarifying.

Reply:
in not active anymore (currently), changed to BottomSheet for now